file_transfer_agent.py

```diff
@@ -346,9 +346,9 @@ def _upload_files_in_parallel(self, file_metas):
                 if len(retry_meta) == 0:
                     # no new AWS token is required
                     break
+                client = self.renew_expired_client()
```

Comment:
hmm,  this renew token happens only if `result_status` is ResultStatus.RENEW_TOKEN?

Reply:
`RENEW_TOKEN` and `RENEW_PRESIGNED_URL`, the line just before breaks if nothing needs to be retried

Reply:
ah, ok